<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myindex</class>
 <widget class="QMainWindow" name="myindex">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>员工考勤管理系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>731</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_baseinfo">
       <property name="text">
        <string>员工基本信息管理</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_workinfo">
       <property name="text">
        <string>员工考勤信息管理</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_exit">
       <property name="text">
        <string>          退出         </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>761</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>员工基本信息:</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>701</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_add">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>380</y>
        <width>96</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_delete">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>380</y>
        <width>96</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_find">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>380</y>
        <width>96</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>查找</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_upadate">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>380</y>
        <width>96</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>卡号:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>180</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>姓名:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>180</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>性别:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cardID">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_name">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>180</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_sex">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>180</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>男</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>女</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>64</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>出生年月:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>64</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>身份证号:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>230</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>电话:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>280</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>住址:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>部门:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>330</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>职位:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>330</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>密码:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>180</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>照片:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_year">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>230</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1990</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1991</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1992</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1993</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1994</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1995</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1996</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1997</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1998</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1999</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2001</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2002</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2003</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2004</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2005</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2006</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2007</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_month">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>230</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>01</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>02</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>03</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>04</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>06</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>06</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>07</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>08</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>09</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>230</y>
        <width>16</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>年</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>230</y>
        <width>16</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>月</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_phone">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>230</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ID">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>280</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_address">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>280</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_department">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>330</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>销售部</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>董事会</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>后勤部</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>技术部</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>其他</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_position">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>330</y>
        <width>89</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>经理</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>员工</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>其他</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>330</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_clear">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>380</y>
        <width>96</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_selectphoto">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>330</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>选择照片</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_photo">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>180</y>
        <width>121</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string>          照片</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pushButton_clear_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>390</y>
        <width>96</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>391</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_find_2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>390</y>
        <width>96</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>查找</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>40</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>卡号:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>90</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>姓名:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>140</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>性别:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>290</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>部门:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>190</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>电话:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>340</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>职位:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>250</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>身份证号:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_starttime">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>起始日期</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_endtime">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>终止日期</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_start">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>110</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_end">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>10</y>
        <width>110</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cardID_2">
      <property name="geometry">
       <rect>
        <x>442</x>
        <y>40</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_name_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>90</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_phone_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>190</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ID_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>250</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_department_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>290</y>
        <width>89</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>董事会</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>技术部</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>销售部</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>后勤部</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_position_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>340</y>
        <width>89</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>经理</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>员工</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_sex_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>140</y>
        <width>89</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>男</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>女</string>
       </property>
      </item>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget_starttime">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>40</y>
        <width>281</width>
        <height>172</height>
       </rect>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget_endtime">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>40</y>
        <width>281</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_org_photo">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>70</y>
        <width>111</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_cur">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>230</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>  刷卡时的照片</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cur_photo">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>250</y>
        <width>111</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_org">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>50</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>   本人照片</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
